sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/models/Checkpoint.java

```diff
@@ -14,7 +14,7 @@
 public class Checkpoint {
```

Comment:
iirc, Ramya wanted to add fullyQualifiedNamespace to this

Reply:
Added. I thought I'll add this in a later PR but included it in this PR itself now.